lib/renderer/src/renderer/core/AbstractCoreRenderer.h

```diff
@@ -0,0 +1,49 @@
+//
+// Created by nikita on 05.12.2021.
+//
+
+#ifndef VANGERS_ABSTRACTCORERENDERER_H
+#define VANGERS_ABSTRACTCORERENDERER_H
+
+#include <cstdint>
+
+#include "../common.h"
+#include "../ResourceId.h"
+
+namespace renderer::core {
+	enum class TextureType {
+		RGBA32,

```

Comment:
A bit of a strange name choice, even if it comes from SDL.
Most modern APIs would call it RGBA8

Reply:
Excatly, it was taken from SD :) I will change to RGBA8 then